fcc-test-suite-template-master/index.html

```diff
@@ -39,11 +39,14 @@ <h1><center>Here's a time line of Dr. César's life:</center></h1>
             Works in the Civilian Conservation Corps, helping starving
             Americans. "I saw how food changed them", he said. "All of this left
             scars on me."
+            <strong></strong>
           </li>
           <li>
             <strong>1937</strong> - Finishes university and takes a job in the
             US Forestry Service
           </li>
+
+          
```

Comment:
pleas remove extra space that make your code un clear and out of proper format😎😎